File: python_stack/Algos/OOP/compareTriplets.py
def compareTriplets(a, b):
    arr=[]
    acount=0
    bcount=0  
    for i in range(len(a)):
        if a[i]>b[i]:
            acount+=1
        elif b[i]>a[i]:
            bcount+=1
    arr.append(acount)
    arr.append(bcount)
    return arr   
a=[2,3,5]
b=[1,3,6]
print(compareTriplets(a,b))

File: python_stack/Algos/Javascript/compareTriplets.js
function compareTriplets(a,b){
    let mySet=[0,0]
    for(let i=0;i<a.length;i++){
        if (a[i]!=b[i])
        {
        if(a[i]>b[i]){
            mySet[0]=mySet[0]+1
        }
        else{
            mySet[1]=mySet[1]+1
        }
    }
        
    }
    console.log(mySet);
    
}
compareTriplets([1, 28, 30],[99, 16, 8])